### PR TITLE
chore!: update env variables to avoid naming misunderstanding (Issue #427)

### PR DESCRIPTION
**Pull request by mikitabut**
### Description
- Issue #427

### CODE REVIEW

**Review by IlyaBondar — DISMISSED** [review]
@mikitabut , need to double check that these variables were updated for review, staging and prod environments

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Review by denys-kolomiitsev — DISMISSED** [review]

**Review by IlyaBondar — APPROVED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Review by denys-kolomiitsev — APPROVED** [review]

File: .github/pr-title-checker-config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "color": "EEEEEE"
   },
   "CHECKS": {
-    "prefixes": ["fix: ", "feat: ", "feature: ", "chore: ", "hotfix: ", "e2e: "]
+    "prefixes": ["fix: ", "feat: ", "feature: ", "chore: ", "hotfix: ", "e2e: ", "fix!: ", "feat!: ", "feature!: ", "chore!: ", "hotfix!: "]
```
Comment (nepalevov) [inline .github/pr-title-checker-config.json:7]: I want to inform you that the exclamation syntax flavor is not supported by the release notes generation script. You still can use it, but PRs with such titles will be treated as _Other_.

**Comment by IlyaBondar** [comment]
/deploy-review 
>Deployment status: success

**Pull request merged**